config/routes.rb

```diff
@@ -0,0 +1,29 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  controller :registration do
+    get 'signup', to: 'registration#new'
```

Comment:
Check if in controller do block, to requires controller name. Also use symbol syntax. `get :signup`